OSDOCS-12075 updating GCP disk types (#82303)
openshift/openshift-docs: modules/installation-configuration-parameters.adoc

      osDisk:
        diskType:
|The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].

> |The default disk type for all machines. Valid values are `pd-balanced`, `pd-ssd`, `pd-standard`, `hyperdisk-balanced`. Control plane machines cannot use the `pd-standard` disk type, so if you specify it here, you must specify a valid disk type using the `controlPlane.platform.gcp.osDisk.diskType` parameter. The default value is `pd-ssd`.

Reviewer: ```suggestion
|The default disk type for all machines. Valid values are `pd-balanced`, `pd-ssd`, `pd-standard`, or `hyperdisk-balanced`. The default value is `pd-ssd`. If you specify a value here, you must specify a valid disk type using the `controlPlane.platform.gcp.osDisk.diskType` parameters. Control plane machines cannot use the `pd-standard` disk type. 
```
I re-wrote the final sentences based on what I think you are trying to say. If it doesn't work, disregard my edits.

Author: Thanks Janelle, I will reword slightly with your suggestion in mind.

      osDisk:
        diskType:
|The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type] for compute machines.

> |Valid values are `pd-ssd`, `pd-standard`, `hyperdisk-balanced`, or `pd-balanced`. The default value is `pd-ssd`.

Reviewer 2: Suggest to list the valid types in the same order as the command `openshift-install explain`, thanks! 
```
$ openshift-install explain installconfig.compute.platform.gcp.osDisk.diskType
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <string>
  Valid Values: "pd-balanced","pd-ssd","pd-standard","hyperdisk-balanced"
  DiskType defines the type of disk. For control plane nodes, the valid values are pd-balanced, pd-ssd, and hyperdisk-balanced.

$
